mission/models.py

```diff
@@ -217,21 +217,35 @@ class MissionOrganization(models.Model):
     remover = models.ForeignKey(get_user_model(), on_delete=models.PROTECT, related_name='remover%(app_label)s_%(class)s_related', null=True, blank=True)
     removed = models.DateTimeField(null=True, blank=True)
 
-    # This is the highest role an organization member can have, depending on their organization role
-    ORGANIZATION_ROLE = (
-        ('M', 'Member'),
-        ('A', 'Admin'),
-    )
-    role = models.CharField(max_length=1, choices=ORGANIZATION_ROLE, default='M')
+    permissions_organization_add = models.BooleanField(default=False)
```

Comment:
I specifically don't want to create roles because that means a large number of roles are needed, and it's confusing to users to work out which permissions those roles have.